app/views/fields/belongs_to/_show.html.erb

```diff
@@ -16,8 +16,12 @@ By default, the relationship is rendered as a link to the associated object.
 %>
 
 <% if field.data %>
-  <%= link_to(
-    field.display_associated_resource,
-    [Administrate::NAMESPACE, field.data],
-  ) %>
+  <% begin %>
+    <%= link_to(
+      field.display_associated_resource,
+      [Administrate::NAMESPACE, field.data],
+    ) %>
+  <% rescue %>
```

Comment:
@zamith I'd say yeah - the function would cover all the cases we're using it for, so it would still be valuable to have. We can always expand the function to support blocks later if we want to.